src/Nav.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './Nav.css';
+
+export default function Nav() {
+  const navStyle = {
+    color: 'blue',
+  };
+  return (
+    <div>
+      <nav>
+        <h1 className="title">Math Magicians</h1>
+        <ul>
+          <Link to="/">
+            <li style={navStyle}>Home</li>
+          </Link>
+          <Link to="/calculator">
+            <li className="middle" style={navStyle}>
+              Calculator
+            </li>
+          </Link>
+          <Link to="/quote">
+            <li style={navStyle}>Quote</li>
+          </Link>
+        </ul>
```

Comment:
- [ ]  I would recommend improving the navbar styling a little bit and make more space between list elements so that it looks good.

![Screenshot from 2021-08-17 21-41-49](https://user-images.githubusercontent.com/22434660/129762757-65959ed6-24c7-4a3c-8814-c22bf1d69d7b.png)